content/docs/get-started/index.md

```diff
@@ -4,13 +4,14 @@ description: 'Learn how you can use MLEM to easily manage and deploy models'
 
 # Get Started
 
-We assume MLEM is already [installed](/doc/install) in your active Python
-environment, as well as `pandas`, `sklearn`, `fastapi`, `uvicorn` and `docker`
-(if not, you can follow the instructions below).
+To get started, we'll start with installing the `mlem` python library containing
+the sdk and cli. For the rest of the guide, we assume MLEM is already
+[installed](/doc/install) in your active Python environment, as well as some
+other useful packages - `pandas`, `sklearn`, `fastapi`, `uvicorn` and `docker`.
 
 <details>
 
-### ⚙️ Expand for setup instructions
+### ⚙️ Expand for set-up instructions
```

Comment:
All these preparatory steps can be hidden (if really needed) in the `<details>` section below. In fact they're all there already (except Docker?) so this seems redundant.